apps/web/pages/index.vue

```diff
@@ -41,7 +41,9 @@
   </div>
   <NuxtLazyHydrate when-visible>
     <div class="max-w-screen-3xl mx-auto md:px-6 lg:px-10 mb-10">
-      <UiMediaCard :image="mediaData.image" :text="mediaData.text" :alignment="mediaData.alignment" />
+      <div v-for="(item, index) in mediaData" :key="index">
```

Comment:
Could you please try: 
  <UiMediaCard
    v-for="(item, index) in mediaData"
    :key="index"
    :image="item.image"
    :text="item.text"
    :alignment="item.alignment"
  />
  
  I would want to avoid having a syntax like <div><div><div>. If the above variant does not work please try looping over a <template>